AllReadyApp/Web-App/AllReady/Views/Activity/Activity.cshtml

```diff
@@ -223,30 +223,18 @@
                                         <span asp-validation-for="SignupModel.AdditionalInfo" class="text-danger"></span>
                                     </div>
                                 </div>
-                                @if (Model.RequiredSkills?.Count > 0)
-                                {
-                                    <div class="form-group">
+                                    <div class="form-group" data-bind="visible: unassociatedSkills.length > 0">

```

Comment:
Okay.  It was deliberate, as I think it aids readability and avoids possible problems with edge cases.  For example, I actually had to think about whether the similar use of the `Description` string field on one of the other `visible` data bindings was intentional or not.  And with the latter there is a potential problem when it's `null`, because both `null == true` and `null == false` are `false` so if KO were not testing with `!value` the code wouldn't work; I realise that the `length` of an array won't ever be null, but I thought it was a good habit.  

Anyway, I've removed the `> 0` and repushed.  I'll try to remember that terseness is highly prized :)